fitnessllm_dataplatform/task_handler.py

```
@@ -1,160 +1,201 @@
+                .collection("stream")
+                .document(data_source.lower())
+                .get()
+                .to_dict()
+            )
+            if strava_user_data is None:
+                raise ValueError(f"User {uid} has no {data_source} data")
+
+            strava_api_interface = StravaAPIInterface(
+                infrastructure_names=self.InfrastructureNames,
+                access_token=self.decryptor(
+                    encrypted_token=strava_user_data["accessToken"]
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)